src/mapml/features/featureGroup.js

```diff
@@ -26,9 +26,13 @@ export var FeatureGroup = L.FeatureGroup.extend({
         this.off("click", this._openPopup);
       }
     }
-
+    
+    L.DomEvent.on(this.options.group, "keyup keydown mousedown", this._handleFocus, this);
```

Comment:
```suggestion
    L.DomEvent.on(this.options.group, "keyup keydown", this._handleFocus, this);
```

Reply:
> If I understand correctly _handleFocus is only handling keyboard events and nothing related to mouse events?

That's what I thought too.

> We can remove mousedown event from the feature group, instead of omitting mouse event on the _handleFocus method.

Yes, I believe that it could be an alternative to fix the bug, and it seems more reasonable than handling the event type inside the _handleFocus function as in the previous commit. Thanks!

Reply:
great, thanks!